release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-security"]
 === Security and compliance
 
+[id="ocp-4-8-security-log-oauth-tokens"]
+==== Audit logging for OAuth access token logout requests
+
+The `Default` audit log policy now logs request bodies for OAuth access token login and logout requests. Previously, only login request bodies were logged.
```

Comment:
@bergerhoffer If I were being picky, I would quibble about _only login request bodies were logged_, as it *could* be interpreted to mean that literally, only requests bodies and nothing else was logged. And suggest: _Previously, logout request bodies were not logged_. 
However, what you have works for release notes.

Reply:
I wasn't super happy with that sentence, so thanks for the good suggestion!